docs/source/motivation.rst

```diff
@@ -0,0 +1,29 @@
+Motivation
+==========
+
+Ensuring data quality is of great importance for many use cases. ``datajudge`` seeks to make this convenient.
+
+Not trying to reinvent the wheel, ``datajudge`` relies on ``pytest`` to test expectations against data. ``datajudge`` allows for the expression of expectations held against data stored in databases. In particular, it allows for comparing different ``DataSource`` s. Yet, it also comes with functionalities to compare data from a single ``DataSource`` to fixed reference values derived from explicit domain knowledge.
+
+
+Comparisons between DataSources
+-------------------------------
+
+The data generating process can be obscure for a variety of reasons. In such scenarios one might ask the questions of
+
+- Has the data 'changed' over time?
+- Was the transformation of the data successful?
+
+In both cases one might want to compare different data - either from different points in time or from different transformation steps - to each other.
```

Comment:
```suggestion
In both cases one might want to compare different data -- either from different points in time or from different transformation steps -- to each other.
```

(Hoping that this renders correctly in Sphinx as –)

Reply:
Hah, funny that the GH diff marks the second `-` green in the first change and the first `-` green in the second change